### ISSUE TITLE
Track script only works for slp files

### ISSUE THREAD

**Issue by sheridana** [talmolab/biogtr#18]
The track [script](https://github.com/talmolab/biogtr/blob/main/biogtr/inference/track.py) currently only works for slp files:

```py
num_videos = len(dataloader.dataset.slp_files)
```

```py
outpath = os.path.join(
            outdir,
            f"{Path(pred_cfg.cfg.dataset.test_dataset.slp_files[i]).stem}_tracking_results",
)
```

I would add a check for video type and add logic to handle non slp_files (e.g for microscopy data).

**Comment by aaprasad** [comment]
yes, the inference script will need to be refactored. There are a lot of changes that have been made since we first wrote it and there are probably better ways to handle it.

**Comment by aaprasad** [comment]
Fixed in #51
